src/html/partials/header.html

```diff
@@ -1,8 +1,18 @@
 <header>
+
   <div class="wrapper--title">
     <h1 class="header__title">A Viget Exploration</h1>
   </div>
-  <div>
-    <img src="../../images/TitleTreatment-01.svg" alt="improving medical adherence">
-  </div>
+
+  <img class="header__img" src="../../images/TitleTreatment-01.svg" alt="improving medical adherence">
+
+  <nav role="navigation" class="nav">
+    <ul class="nav__list">
+      <li>The Problem</li>
+      <li>Key Factors</li>
+      <li>The Question</li>
+      <li>The Solution</li>
```

Comment:
For accessibility, these would need to be in page anchors, e.g.
```
<li><a href="#solution">The Solution</a></li>
```
and that would link down to "the solution" section (id attribute) further down the page...
```
<section id="solution">
  ...
</section>
```